lib/services/zohoprojects.rb

```diff
@@ -0,0 +1,55 @@
+class Service::ZohoProjects < Service::Base
+  title 'Zoho Projects'
+
+  string :project_id,
+    :label => "Note: You need a ZohoProjects enterprise account to enable this integration. " \
+    "To get your Project ID and Auth Token, head to your ZohoProject's dashboard and look under \"Service Hooks\". " \
+    "If you don't see \"Service Hooks\" on your dashboard, you can enable this panel under \"Project Customization\"." \
+    "<br /><br />" \
+    "Project ID"
+
+  string :authtoken, :label => 'Auth Token'
+
+  page 'Project Information', [:project_id, :authtoken]
+
+  def receive_issue_impact_change(config, issue)
+    payload = JSON.generate(:event => 'issue_impact_change', :payload => issue)
+
+    response = send_request_to_projects config, payload
+    if response.status != 200
+      raise "Problem while sending request to Zoho Projects, Status: #{response.status}, Body: #{response.body}"
+    end
+
+    return { :zohoprojects_bug_id => response.body }
```

Comment:
Yup, this gets stored in the issue record under the field `sr`.  Right now it's unused but it's there for future bi-directional integration.